netsim/augment/links.py

```diff
@@ -387,6 +387,11 @@ def assign_link_prefix(
       addr_pools: Box,
       nodes: Box,
       link_path: str = 'links') -> Box:
+  
+  # Don't assign a prefix to physical links that are part of a lag
+  if 'lag' in link and link.get("type","")!="lag":
```

Comment:
This should be handled with 'ipv4: False' and 'ipv6: False' in the LAG module. There's no need to tweak generic link augmentation if we have other mechanisms.

Reply:
I added ```prefix: False```